client/signup/steps/pressable-store/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
You may prefer the following syntax

```js
import React, { PropTypes } from 'react';
```

which allows `propTypes` declarations to refer directly to `PropTypes`.